code/modules/mining/mine_turfs.dm

```diff
@@ -149,6 +150,8 @@ turf/unsimulated/mineral/ChangeTurf(var/turf/N, var/tell_universe=1, var/force_l
 
 
 /turf/unsimulated/mineral/ex_act(severity)
+	if(severity < mining_difficulty)
```

Comment:
Shouldn't this sign be the other way around? The lower severity is, the more powerful the explosion